lib/verifyLicenseCheck.ts

```diff
@@ -0,0 +1,16 @@
+import crypto from 'crypto';
+
+type VerifyLicenseCheckArgs = {
+  publicKey: string
+  signature: string
+  payload: string
+}
+
+function verifyLicenseCheck({ publicKey, signature, payload }: VerifyLicenseCheckArgs) {
```

Comment:
We also need to add an `export` for this in the `lib/index.ts` file (and add it to the window object) so it's included in the package output.